### PR TITLE
Grant/ginkgo

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
Ginkgo versions of Open edX docs

### CODE REVIEW

File: en_us/install_operations/source/platform_releases/ginkgo.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* edx-analytics-data-api
+* edx-analytics-pipeline
+* edx-certificates
+* edx-custom-a11y-rules
+* edx-demo-course
+* edx-documentation
+* edx-notes-api
+* edx-ui-toolkit
+* notifier
+* programs
```
Comment (nedbat) [inline en_us/install_operations/source/platform_releases/ginkgo.rst]: Remove programs.

File: en_us/install_operations/source/platform_releases/ginkgo.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+****************************
+
+A git tag identifies the version of Open edX code that is the Ginkgo release.
+You can find the most up-to-date git tag for the current Open edX release on
+the `Open edX Releases Wiki page`_.
+
+The following Open edX git repositories have the Ginkgo git tag:
+
+* edx-platform
+* configuration
```
Comment (nedbat) [inline en_us/install_operations/source/platform_releases/ginkgo.rst:32]: Add "course_discovery"

**Review by nedbat — DISMISSED** [review]

**Comment by grantgoodmanedX** [comment]
@nedbat I added a migration section in this PR, rather than the release note PR, since they are on different branches. I have a [RUN SOME SCRIPT] placeholder that might be where you add your migration script and instructions. I'll also need information about how the component migration steps fit in with an overall migration script, if at all.

**Comment by nedbat** [comment]
@feanil @jibsheet Would you mind taking a quick look at the upgrade instructions? I did a bunch of bash to make dumping the database smooth.

**Comment by jibsheet** [comment]
@nedbat This looks good to me, @feanil has a few questions that cover both of us.

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@nedbat I had some questions around mongo indexes. If we add or update indexes between two releases, we would want to point them to docs for the changed indexes. I don't think that happened here but just making sure.  Also, by default our code doesn't create indexes automatically on mongo, do we need to add docs for how to create the default indexes here as well?

**Comment by nedbat** [comment]
@feanil @jibsheet I'm looking to you for expertise on this.  If you say we need to create the default indexes, then let's add those instructions.

File: en_us/install_operations/source/platform_releases/ginkgo.rst [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+not normally use the ``guest_email`` column. If your installation does not use
+this column, you can avoid the table lock by using the ``--fake`` argument in
+migrating the ``orders`` table, running the migrate command in this form:
+
+.. code-block:: bash
+
+   ./manage.py migrate orders 0013 --fake
+
+=====================
+Upgrading Celery
```
Comment (grantgoodmanedX) [inline en_us/install_operations/source/platform_releases/ginkgo.rst]: @nedbat Can we drop this section? I think it's covered by the migration instructions above.

Comment (nedbat) [reply en_us/install_operations/source/platform_releases/ginkgo.rst]: Oh, right. I meant to do that, sorry.  Drop this section.

**Comment by catong** [comment]
@grantgoodmanedX Looks good; I built each Gingko version guide included in this PR and each one built successfully, looked fine, but 2 of the guides have unrelated build errors that should clear after you rebase against master. 👍

**Pull request merged**